### PR TITLE
Beef up the CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by thockin**
Add links to CNCF CLA, as well as links to file an issue and to the help-wanted
list.

@kubernetes/contributor-experience

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35544)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,48 +4,101 @@ Want to hack on kubernetes? Yay!
...
 ### Contributor License Agreements
 
-We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+We'd love to accept your patches! Before we can take them, we have to jump a
+couple of legal hurdles.
 
-Please fill out either the individual or corporate Contributor License Agreement (CLA).
+Please fill out either the individual or corporate Contributor License
+Agreement (CLA).  As of Q4 2016, we're transitioning from Google's CLA to the
+CNCF CLA.
```
Comment (grodrigues3) [inline CONTRIBUTING.md]: super minor comment, but maybe expand the acronym for its first usage

**Review by grodrigues3 — COMMENTED** [review]
This is great!  I think the new contributor ramp-up was poorly documented, and this certainly helps!  LGTM.

**Comment by thockin** [comment]
Done

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,48 +4,101 @@ Want to hack on kubernetes? Yay!
...
+We have a [Developer's Guide](docs/devel/README.md) that outlines everything
+you need to know from setting up your dev environment to how to get faster Pull
+Request reviews. If you find something undocumented or incorrect along the way,
+please feel free to send a Pull Request.
 
 ## Filing issues
 
-If you have a question about Kubernetes or have a problem using it, please read the [troubleshooting guide](docs/troubleshooting.md) before filing an issue.
+If you have a question about Kubernetes or have a problem using it, please
+start with the [troubleshooting guide](docs/troubleshooting.md).  If that
```
Comment (bgrant0607) [inline CONTRIBUTING.md]: The actual troubleshooting guide has moved to kubernetes.io. We should delete the one in this repo.

**Comment by grodrigues3** [comment]
didn't realize it had already gotten merged.  was removing /lgtm so you could get brian's fix in

**Pull request merged**